ecosystem.rst

```diff
@@ -47,7 +47,7 @@ Example Apps
 External Notification
 ---------------------
 
-These are PostgreSQL bridges that propagate LISTEN/NOTIFY to external queues for further processing. This allows stored procedures to initiate actions outside the database such as sending emails.
+These are PostgreSQL bridges that propagate LISTEN/NOTIFY to external queues for further processing. This allows for user-designed functions to initiate actions outside the database such as sending emails.
```

Comment:
Not sure if this references any stored procedure (like the ones used on triggers), or also to the user-defined functions exposed by PostgREST, or both.

Reply:
Refers to internal functions - not exposed by PostgREST.